webots_ros2_universal_robot/launch/multirobot_launch.py

```diff
@@ -74,27 +74,31 @@ def generate_launch_description():
         executable='spawner.py',
         output='screen',
         prefix=controller_manager_prefix,
+        namespace='ur5e',
         arguments=['ur_joint_trajectory_controller', '-c', 'ur5e/controller_manager'] + controller_manager_timeout,
     )
     ur5e_joint_state_broadcaster_spawner = Node(
         package='controller_manager',
         executable='spawner.py',
         output='screen',
         prefix=controller_manager_prefix,
+        namespace='ur5e',
         arguments=['ur_joint_state_broadcaster', '-c', 'ur5e/controller_manager'] + controller_manager_timeout,
     )
     abb_trajectory_controller_spawner = Node(
         package='controller_manager',
         executable='spawner.py',
         output='screen',
         prefix=controller_manager_prefix,
+        namespace='abb',
         arguments=['abb_joint_trajectory_controller', '-c', 'abb/controller_manager'] + controller_manager_timeout,
     )
     abb_joint_state_broadcaster_spawner = Node(
         package='controller_manager',
         executable='spawner.py',
         output='screen',
         prefix=controller_manager_prefix,
+        namespace='abb',
```

Comment:
```suggestion
        arguments=['ur_joint_trajectory_controller', '-c', 'ur5e/controller_manager'] + controller_manager_timeout,
    )
    ur5e_joint_state_broadcaster_spawner = Node(
        package='controller_manager',
        executable='spawner.py',
        output='screen',
        prefix=controller_manager_prefix,
        arguments=['ur_joint_state_broadcaster', '-c', 'ur5e/controller_manager'] + controller_manager_timeout,
    )
    abb_trajectory_controller_spawner = Node(
        package='controller_manager',
        executable='spawner.py',
        output='screen',
        prefix=controller_manager_prefix,
        arguments=['abb_joint_trajectory_controller', '-c', 'abb/controller_manager'] + controller_manager_timeout,
    )
    abb_joint_state_broadcaster_spawner = Node(
        package='controller_manager',
        executable='spawner.py',
        output='screen',
        prefix=controller_manager_prefix,
```
These namespaces are not needed as those are just controller spawners